jobserver/views/job_requests.py

```diff
@@ -160,16 +170,10 @@ def get_context_data(self, **kwargs):
         }
 
     def get_form_kwargs(self):
-        kwargs = super().get_form_kwargs()
-        kwargs["actions"] = [a["name"] for a in self.actions]
-
-        # get backends from the current user
-        backends = Backend.objects.filter(
-            memberships__in=self.request.user.backend_memberships.all()
-        )
-        kwargs["backends"] = backends_to_choices(backends)
-
-        return kwargs
+        return super().get_form_kwargs() | {
```

Comment:
This is much nicer. The pipe operator is a new one for me and I like the usage of it here